styles.css

```diff
@@ -478,11 +485,209 @@ input {
+  .start {
+    background: url("images/Header-llustration-desktop.svg");
+    background-repeat: no-repeat;
+    background-size: 110%;
+    background-position: top -60%;
+    height: 90%;
+  }
+
+  /* nav-bar */
+  .menu {
+    display: none;
+  }
+
+  .navigation {
+    display: inline;
+    width: 100%;
+    font-weight: 700;
+    color: #172b4d;
+  }
+
+  .desktop-nav {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+  }
+
+  .desktop-nav li {
+    margin: 10px;
+  }
+
+  /* banner */
+  .banner {
+    margin-top: 8%;
+    margin-bottom: 0%;
+  }
+
+  .banner h1 {
+    font-size: 40px;
+    width: 690px;
+    margin-left: 5%;
+  }
+
+  .banner p {
+    width: 570px;
+    margin-left: 13%;
+    padding-top: 2%;
+  }
+
+  /* social-media */
+  .social-media ul {
+    position: absolute;
+    left: 50%;
+    right: 50%;
+  }
+
+  /* indicator */
+  .indicator {
+    display: none;
+  }
+
+  /* recent-works */
+  .topper {
+    position: relative;
+    margin-bottom: 0%;
+  }
+
+  .recent-works {
+    margin-top: 3%;
+  }
+
+  .recent-works-cards {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    gap: 24px;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 0;
+    height: 988px;
+    width: auto;
+  }
+
+  .works {
+    width: 30.33%;
+  }
+
+  .language-tags {
+    display: flex;
+    flex: 1;
+  }
+
+  .language-tags li {
+    font-weight: 600;
+    font-size: 10px;
+    color: #3a4a42;
+    padding: 1px 3px;
+    background: #ebf0ee;
+    border-radius: 4px;
+  }
+
+  .work-details h3 {
+    padding-top: 20px;
+    font-size: 22px;
+  }
+
+  /* about-me */
+  .about-me-text {
+    background-image: url("images/top-tail.svg"), url("images/abtme.svg");
+    background-size: 80px;
+    background-position: right 50%, left 70%;
+    align-items: center;
+    height: 300px;
+    width: 100%;
+  }
+
+  #about-me-para {
+    width: 550px;
+    font-size: 17px;
+  }
+
+  .about-me-card {
+    box-sizing: border-box;
+    padding: 1px;
+    height: 400px;
+    display: flex;
+    flex-direction: row;
+    gap: 16px;
+  }
+
+  /* languages frameworks skills */
+  .lang-card {
+    width: 33%;
+    height: auto;
+  }
+
+  .framework-card {
+    width: 33%;
+    height: auto;
+  }
+
+  .skills-card {
+    width: 33%;
+    height: auto;
+  }
+
+  /* form */
+  form {
+    height: 330px;
+  }
+
+  .form {
+    background:
+      url(images/kando-left.svg) left 99%/15%,
+      url(images/kando.svg) right 87%/10%,
+      url(images/rock.svg) 20% 90%/10%;
+    background-repeat: no-repeat;
+    display: grid;
+    grid-template-columns: 370px auto;
+    align-items: center;
+    height: 80%;
+    position: relative;
+  }
+
+  .form h2 {
+    font-size: 26px;
+    text-align: start;
+    align-items: center;
+    line-height: 42px;
+    margin-top: -30px;
+  }
+
+  .form-inputs {
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
- [ ] Kindly add the correct on-focus effect on your form inputs. Please note from the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=h5L9p23oW5s1BP0W-0) that all the inputs' borders should be `#d0d9d4` but changes to `#60c095` on focus.